biapol_utilities/__init__.py

```diff
@@ -16,7 +16,8 @@
 __version__ = "0.0.1"
 
 # from .measure import *
-from .label import *
+from . import label
```

Comment:
I don't think this is necessary as the subfolder is called "label" anyway. Would you mind trying it without that line?

Reply:
> I don't think this is necessary as the subfolder is called "label" anyway. Would you mind trying it without that line?

For me it seems like it is necessary. Commenting the line throws an error that the subpackage `label` is not found when I run the respective test.